### PR TITLE
Additional Cubeviz docs sections

### PR DESCRIPTION
**Pull request by rosteen**
Fills in some more sections of the Cubeviz docs.

### CODE REVIEW

File: docs/cubeviz/displaycubes.rst [hunk 1]
```diff
@@ -27,11 +29,18 @@ also be collapsed into a spectrum using the same statistic and displayed in
...
 data in each spectral slice.
 
+.. _cubeviz-selecting-data:
 
 Selecting Data Set
 ==================
 
-Coming soon
+The data to be displayed in each viewer can be selected and de-selected by 
+clicking on the "hammer and screwdriver" icon at the top left of each viewer.
```
Comment (javerbukh) [inline docs/cubeviz/displaycubes.rst:38]: Patrick suggested using :guilabel:\`Hammer and screwdriver\` to describe UI elements.

File: docs/cubeviz/displaycubes.rst [hunk 2]
```diff
@@ -27,11 +29,18 @@ also be collapsed into a spectrum using the same statistic and displayed in
...
 
+.. _cubeviz-selecting-data:
 
 Selecting Data Set
 ==================
 
-Coming soon
+The data to be displayed in each viewer can be selected and de-selected by 
+clicking on the "hammer and screwdriver" icon at the top left of each viewer. 
+Then, click the "gear" icon to access the "Data" tab. Here, you can click the
```
Comment (javerbukh) [inline docs/cubeviz/displaycubes.rst:39]: Same as above.

File: docs/cubeviz/import_data.rst [hunk 3]
```diff
@@ -1,5 +1,43 @@
...
+attempt to automatically parse the data into the viewers as described in 
+:ref:`cubeviz-display-cubes`. Cubeviz also supports loading FITS files that can
+be parsed as :class:`~specutils.Spectrum1D`, which will be loaded into the 
+1D spectrum viewer.
+
+Importing data through the GUI
+------------------------------
+
+The first way that users can load their data into the Cubeviz application is 
+by clicking the "Import" button at the top left of the application's user interface.
```
Comment (javerbukh) [inline docs/cubeviz/import_data.rst:19]: Same as above.

File: docs/cubeviz/import_data.rst [hunk 4]
```diff
@@ -1,5 +1,43 @@
...
+
+The first way that users can load their data into the Cubeviz application is 
+by clicking the "Import" button at the top left of the application's user interface. 
+This opens a dialogue where the user can enter the path of file that can be parsed 
+as a :class:`~spectral_cube.SpectralCube`
+or :class:`~specutils.Spectrum1D` in the text field:
+
+.. image:: img/cubeviz_import_data.png
+
+After clicking "Import", the data file will be parsed and loaded into the
```
Comment (javerbukh) [inline docs/cubeviz/import_data.rst:26]: Here as well.

File: docs/cubeviz/import_data.rst [hunk 5]
```diff
@@ -1,5 +1,43 @@
...
+by clicking the "Import" button at the top left of the application's user interface. 
+This opens a dialogue where the user can enter the path of file that can be parsed 
+as a :class:`~spectral_cube.SpectralCube`
+or :class:`~specutils.Spectrum1D` in the text field:
+
+.. image:: img/cubeviz_import_data.png
+
+After clicking "Import", the data file will be parsed and loaded into the 
+application. A notification will appear to let users know if the data import 
+was successful. Afterward, the new data set can be found in the "Data" tab
```
Comment (javerbukh) [inline docs/cubeviz/import_data.rst:28]: And here.

**Review by javerbukh — APPROVED** [review]
Left a couple comments but otherwise looks good!

File: docs/cubeviz/displaycubes.rst (hunk 1 above)
Comment (rosteen) [reply docs/cubeviz/displaycubes.rst:38]: Good call, will push an update soon

**Review by PatrickOgle — APPROVED** [review]

**Comment by rosteen** [comment]
I just rebased to the traitlets fix on master to (hopefully) get the tests passing, and squashed most of the commits for a cleaner history. Should be good to merge once the tests pass.

**Comment by rosteen** [comment]
Everything passing but the know Windows CI problem

**Comment by javerbukh** [comment]
Merging.

**Pull request merged**